icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/PluralRulesTest.java

```diff
@@ -399,7 +399,10 @@ public void testUniqueRules() {
                             }
                             String old = collisionTest.get(item);
                             if (old != null) {
-                                errln(locale + "\tNon-unique rules: " + item + " => " + old + " & " + foundKeyword);
+                                if (!locale.getLanguage().equals("fr") ||
+                                        !logKnownIssue("21328", "fr Non-unique rules: 1e6 => one & many0")) {
```

Comment:
Is that a typo? many0 --> many?
(Could use `git commit --amend` to fix it and keep it to a single commit.)